src/command/schema/push.rs

```diff
@@ -0,0 +1,70 @@
+use crate::client::get_rover_client;
+use anyhow::Result;
+use rover_client::query::schema::push;
+use std::path::{Path, PathBuf};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct Push {
```

Comment:
have we decided to omit federated graphs for now?

Reply:
Nope! That's handled in the `service push` command instead of here :)